### DISCUSSION TITLE
Не выполняются скрипты в cgi-bin. Exec format error

### DISCUSSION THREAD

**Discussion by qk4l** [Arkady23/V380-Web#45] [General]
Добрый день,

Спасибо за WEB сервер!

К сожалению на V380E4_C3 столкнулся с тем, что httpd не может выполнять cgi скрипты.
Из-за этого соответсвенно в WEB ничего не работает.

Может сталкивались с этим?

```
/mnt/mtd # cat mvconf/patchmanage.conf 
[PATCH]
patchmd5=
downloadintmp=1
hwname=V380E4_C3
wifitype=8
kertype=31
version=1010603
nextaction=0
updateready=1


/mnt/mtd # /mnt/sdcard/bin/httpd  -p 80 -h /mnt/sdcard/ -f -vvvvv

192.168.94.8:51868: connected
192.168.94.8:51868: url:/cgi-bin/test.sh
192.168.94.8:51868: can't execute 'test.sh': Exec format error
192.168.94.8:51868: response:404
192.168.94.8:51868: closed
192.168.94.8:51868: closed
^C^Z[1]+  Stopped                    /mnt/sdcard/bin/httpd -p 80 -h /mnt/sdcard/ -f -vvvvv
/mnt/mtd # /mnt/sdcard/cgi-bin/test.sh 
Content-Type: text/html

        test    /mnt/mtd # echo $?
0
```

Пробовал заменить httpd из поставки бинарём из https://busybox.net/downloads/binaries/1.21.1/busybox-armv7l но результат аналогичный - `Exec format error`.

**Comment by Arkady23** [comment]
Может что-то с правами? Например, sd карта не в FAT32, а в другом формате.

**Reply by Arkady23** [reply]
Может другие библиотеки поставить? Возьмите из моего Линукса, замените на карте в папке bin. Кстати в обновлении уже есть ftp и он у вас должен работать.
[bin.zip](https://github.com/Arkady23/V380-Web/files/9276742/bin.zip)
